Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppAcquireTokenViewController.m

```diff
@@ -704,4 +730,89 @@ - (IBAction)changeProfile:(id)sender
     [self.navigationController pushViewController:[ADTestAppProfileViewController sharedProfileViewController] animated:YES];
 }
 
+- (void)showClaimsPickerView
+{
+    self.claimsPickerSheet = [UIAlertController alertControllerWithTitle:@"Select claim" message:nil preferredStyle:UIAlertControllerStyleActionSheet];
```

Comment:
nit: Can this code be moved to a separate controller? This class is growing fast :)

Reply:
done